photongo/Readme.md

```diff
@@ -122,356 +154,484 @@ enum Role {
 }
 ```
 
-## Reading Data
+### Basic information
+
+This spec assumes the following decisions:
 
-The read examples follow along closely with the documentation here: https://www.prisma.io/docs/prisma-client/basic-data-access/reading-data-GO-go05/
+- Fluent API (vs. structs API)
+- Null-structs for nullability (vs. using pointers) [photon-go#1](https://github.com/prisma/photongo/issues/1)
+- Require passing a context explicitly [photon-go#6](https://github.com/prisma/photongo/issues/6)
+- Generate code into a single file [photon-go#3](https://github.com/prisma/photongo/issues/3)
 
-### Find
+### Generated models
 
-For example, the following query returns all scalar fields of a single User
+The above data model will generate the following Go models.
 
 ```go
-usr, err := client.User.Find(user.Where().Email("ada@prisma.io"))
+type Post struct {
+  Id        string     `json:"id"`
+  CreatedAt time.Time  `json:"createdAt"`
+  UpdatedAt time.Time  `json:"updatedAt"`
+  Title     string     `json:"title"`
+  Desc      NullString `json:"desc"`
```

Comment:
Can you provide a full working snippet? It's hard to know where these types are coming from.